src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -214,16 +282,18 @@
                    id="path"
                    data-testid="rgw_path"
                    formControlName="path"
-                   [ngbTypeahead]="bucketDataSource">
+                   [ngbTypeahead]="bucketDataSource"
+                   (selectItem)="pathChangeHandler()"
+                   (blur)="pathChangeHandler()">
 
             <span class="invalid-feedback"
                   *ngIf="nfsForm.showError('path', formDir, 'required')"
                   i18n>This field is required.</span>
             <span class="invalid-feedback"
                   *ngIf="nfsForm.showError('path', formDir, 'bucketNameNotAllowed')"
                   i18n>The bucket does not exist or is not in the default realm (if multiple realms are configured).
-                       To continue, <a routerLink="/rgw/bucket/create"
-                                       class="btn-link">create a new bucket</a>.</span>
+                      To continue, <a routerLink="/rgw/bucket/create"
+                                      class="btn-link">create a new bucket</a>.</span>
```

Comment:
lets add i18n

Reply:
I think it's already there 
```<span class="invalid-feedback"
                  *ngIf="nfsForm.showError('path', formDir, 'bucketNameNotAllowed')"
                  i18n>The bucket does not exist or is not in the default realm (if multiple realms are configured).
                      To continue, <a routerLink="/rgw/bucket/create"
                                      class="btn-link"
                                      i18n>create a new bucket</a>.</span>
```

-   And if I try to add it inside <a> it shows 
`Cannot mark an element as translatable inside of a translatable section. Please remove the nested i18n marker.`

Reply:
Okay then cool, thanks!